client/tailwind.config.js

```diff
@@ -63,6 +68,23 @@ module.exports = {
     },
     spacing,
   },
-  plugins: [typeScale],
-  corePlugins: {},
+  plugins: [
+    typeScale,
+    vanillaRTL,
+    /**
+     * forced-colors media query for Windows High-Contrast mode support
+     * See:
+     * - https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors
+     * - https://github.com/tailwindlabs/tailwindcss/blob/7b4cc36f5ea1f7e208dcd3af3e8288878735f45f/src/corePlugins.js#L169-L172
```

Comment:
Minor nitpick, instead of pinning to a commit, it is easier to visually parse the URL when pinned to a branch, should not change over time and easier to see what we are linking to.

```suggestion
     * - https://github.com/tailwindlabs/tailwindcss/blob/v3.0.23/src/corePlugins.js#L168-L171
```